app/src/main/java/com/janewaitara/businesscardapp/BusinessCard.kt

```diff
@@ -0,0 +1,10 @@
+package com.janewaitara.businesscardapp
```

Comment:
Nice idea incorporating a class and making multiple contact cards! Very creative.